index.js

```diff
@@ -39,6 +40,8 @@ export default class QRCodeScanner extends Component {
       PropTypes.string,
     ]),
     notAuthorizedView: PropTypes.element,
+    permissionDialogTitle: PropTypes.string,
+    permissionDialogMessage: PropTypes.string,
```

Comment:
can we also add these two props in the default props object and pass in some standard strings, this would be important to ensure backwards compatibility (since missing these props when the component is rendered will currently cause an error to be thrown on line 88/89, thank!